src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -66,25 +71,43 @@ const UserInformationContent: React.FunctionComponent<PropsType> = ({
     }
   }, [profile, user]);
 
+  const handlerChangeStatus = useCallback(
+    (value: string) => {
+      if (userWithId) {
+        updateUserOnlineStatus({
+          status: value,
+          userId: userWithId?.id,
+        });
+      }
+    },
+    [userWithId]
+  );
+
   if (!user || !userWithId) return null;
 
+  const profileStatus = profile?.status ?? "";
+
```

Comment:
move this into `useProfileStatus` hook. It will return all the handlers + current status

Reply:
@mike-lvov instead of `useCallback`, would this be better using `useAsyncFn`?

- https://github.com/streamich/react-use/blob/master/docs/useAsyncFn.md

Reply:
@0xdevalias no, I think that hook works better for that case